src/plugins/method.ts

```diff
@@ -8,8 +8,6 @@ export default (yargs: Argv) => {
   });
 
   return (config: AlphaCliConfig, { request }: AlphaCliArguments) => {
-    if (request) {
-      config.method = request;
-    }
+    config.method = request ?? 'GET';
```

Comment:
What was happening when we weren't setting a method?

Reply:
I'm pretty sure it was defaulting to a GET.  I was having issues with the proxy reporting a 404, so this was part of the debugging.  I can remove it.